packages/visx-brush/src/BaseBrush.tsx

```diff
@@ -10,9 +10,10 @@ import { MarginShape, Point, BrushShape, ResizeTriggerAreas, PartialBrushStartEn
 
 const BRUSH_OVERLAY_STYLES = { cursor: 'crosshair' };
 
-type MouseHandlerEvent =
+type PointerHandlerEvent =
```

Comment:
I think all of this can be simplified to `React.PointerEvent<SVGRectElement>`, which is already equivalent to `MouseEvent | TouchEvent`